plugins/language_edp.lua

```diff
@@ -0,0 +1,633 @@
+    ["ratio"] = "keyword",
+    ["rescaling"] = "keyword",
+    ["splitin2"] = "keyword",
+    ["splitpbedge"] = "keyword",
+    ["thetamax"] = "keyword",
+    ["uniform"] = "keyword",
+    ["fixedborder"] = "keyword",
+    ["flags"] = "keyword",
+    ["ivalue"] = "keyword",
+    ["maxit"] = "keyword",
+    ["mode"] = "keyword",
+    ["ncv"] = "keyword",
+    ["nev"] = "keyword",
+    ["rawvector"] = "keyword",
+    ["sigma"] = "keyword",
+    ["sym"] = "keyword",
+    ["tol"] = "keyword",
+    ["vector"] = "keyword",
+    ["which"] = "keyword",
+    ["op"] = "keyword",
+    ["t"] = "keyword",
+    ["eps"] = "keyword",
+    ["nbiter"] = "keyword",
+    ["precon"] = "keyword",
+    ["veps"] = "keyword",
+    ["tgv"] = "keyword",
+    ["tolpivot"] = "keyword",
+    ["meditff"] = "keyword",
+    ["save"] = "keyword",
+    ["orientation"] = "keyword",
+    ["ptmerge"] = "keyword",
+    ["transfo"] = "keyword",
+    ["optimize"] = "keyword",
+    ["aspectratio"] = "keyword",
+    ["bb"] = "keyword",
+    ["boundary"] = "keyword",
+    ["bw"] = "keyword",
+    ["cut"] = "keyword",
+    ["grey"] = "keyword",
+    ["hsv"] = "keyword",
+    ["nbarrow"] = "keyword",
+    ["ps"] = "keyword",
+    ["varrow"] = "keyword",
+    ["viso"] = "keyword",
+    ["init"] = "keyword",
+    ["strategy"] = "keyword",
+    ["tolpivotsym"] = "keyword",
+    ["facetcl"] = "keyword",
+    ["holelist"] = "keyword",
+    ["nboffacetcl"] = "keyword",
+    ["nbofregions"] = "keyword",
+    ["regionlist"] = "keyword",
+    ["switch"] = "keyword",
+    ["refface"] = "keyword",
+    ["split"] = "keyword",
+    ["zbound"] = "keyword",
+    ["labeldown"] = "keyword",
+    ["labelmid"] = "keyword",
+    ["labelup"] = "keyword",
+    ["opt"] = "keyword",
+    ["mpiMAX"] = "keyword",
+    ["mpiMIN"] = "keyword",
+    ["mpiSUM"] = "keyword",
+    ["mpiPROD"] = "keyword",
+    ["mpiLAND"] = "keyword",
+    ["mpiLOR"] = "keyword",
+    ["mpiLXOR"] = "keyword",
+    ["mpiBAND"] = "keyword",
+    ["mpiBXOR"] = "keyword",
+    ["border"] = "keyword2",
+    ["Cmapmatrix"] = "keyword2",
+    ["Cmatrix"] = "keyword2",
+    ["complex"] = "keyword2",
+    ["fespace"] = "keyword2",
+    ["func"] = "keyword2",
+    ["ifstream"] = "keyword2",
+    ["mapmatrix"] = "keyword2",
+    ["matrix"] = "keyword2",
+    ["mesh"] = "keyword2",
+    ["mesh3"] = "keyword2",
+    ["ofstream"] = "keyword2",
+    ["problem"] = "keyword2",
+    ["real"] = "keyword2",
+    ["solve"] = "keyword2",
+    ["string"] = "keyword2",
+    ["varf"] = "keyword2",
+    ["macro"] = "keyword2",
+    ["dmatrix"] = "keyword2",
+    ["adj"] = "function",
+    ["find"] = "function",
+    ["rfind"] = "function",
+    ["seekg"] = "function",
+    ["tellg"] = "function",
+    ["AddLayers"] = "function",
+    ["AffineCG"] = "function",
+    ["AffineGMRES"] = "function",
+    ["BFGS"] = "function",
+    ["EigenValue"] = "function",
+    ["LinearCG"] = "function",
+    ["LinearGMRES"] = "function",
+    ["NLCG"] = "function",					
```

Comment:
Whitespace at the end.

Reply:
I think you were referring to the empty line at the end of the function list? if yes now it solved.